fuzzing/private/oss_fuzz/repository.bzl

```diff
@@ -103,12 +103,37 @@ def _extract_build_params(
         instrum_cxxopts = instrum_cxxopts,
     )
 
+# The paths under $OUT of the OSS-Fuzz provided Jazzer files.
+_JAZZER_IN_AGENT_DEPLOY_JAR = "jazzer_agent_deploy.jar"
+_JAZZER_IN_DRIVER = "jazzer_driver"
+_JAZZER_IN_DRIVER_ADDRESS_SANITIZER = "jazzer_driver_asan"
+
+# The filenames under which the OSS-Fuzz provided Jazzer files will be available
+# in this repository.
+_JAZZER_OUT_AGENT_DEPLOY_JAR = _JAZZER_IN_AGENT_DEPLOY_JAR
+_JAZZER_OUT_DRIVER_NO_SANITIZER = "jazzer_driver_no_sanitizer"
+_JAZZER_OUT_DRIVER_WITH_SANITIZER = "jazzer_driver_with_sanitizer"
```

Comment:
If there is an intuitive connection between the `*_IN_*` and `*_OUT_*` forms, I think it would be more visible if the variables were grouped by name, e.g.:

```
_JAZZER_IN_AGENT_DEPLOY_JAR = ...
_JAZZER_OUT_AGENT_DEPLOY_JAR = ...

_JAZZER_IN_DRIVER = ...
_JAZZER_IN_DRIVER_ADDRESS_SANITIZER = ...
_JAZZER_OUT_DRIVER_NO_SANITIZER = ...
_JAZZER_OUT_DRIVER_WITH_SANITIZER = ...
```

Reply:
I grouped them as you suggested, but also added further comments. Let me know if you find this too verbose.

Reply:
These are useful, thanks!